tests/functional/pos/test_pos.py

```diff
@@ -32,7 +32,7 @@ def override_generated_fields(record):
 
 
 @pytest.fixture(scope="function")
-def set_up_oai_environment():
+def set_up_environment():
```

Comment:
This is actually a bad name, using a verb `set_up` hints that the function does something (has side effects).
In this case, it should probably be called something like `configuration` or similar, as what it does, is give back a configuration.

Another option is to have two fixtures instead of one, one with the wait, and the other with the config.